gm4_mountaineering/data/gm4_mountaineering/functions/ski_effects/downhill_ski_speed/add_downhill_ski_speed.mcfunction

```diff
@@ -0,0 +1,8 @@
+# Add increased speed when player is on snowy block, wearing skis, and travelling downhill (>=2 blocks descended)
+# @s = player without skiing tag and having fallen >=2 blocks
+# at @s
+# run from main
+
+tag @s add gm4_mountaineering_skiing_downhill
+scoreboard players set @s gm4_mountaineering_ski_downhill 4
+attribute @s minecraft:generic.movement_speed modifier add 42a046eb-1425-4efe-831b-f891d8ede18f gm4_mountaineering_downhill_speed_boost 0.150 multiply_base
```

Comment:
I won't leave this comment on other attribute commands, but you should look through the rest and fix them as well
```suggestion
attribute @s minecraft:generic.movement_speed modifier add 42a046eb-1425-4efe-831b-f891d8ede18f "gm4_mountaineering_downhill_speed_boost" 0.150 multiply_base
```